gdal/configure.ac

```
@@ -1243,6 +1261,7 @@ else
     fi
   else
     ORIG_LIBS="$LIBS"
+    proj_libdir=$with_proj/lib
```

At least for consistence with similar practice in that file
```suggestion
    proj_libdir="$with_proj/lib"
```
(to adapt in similar cases)